lib/index.js

```diff
@@ -2,5 +2,6 @@ const requireindex = require('requireindex');
 
 module.exports.rules = requireindex(__dirname + '/rules');
 module.exports.configs = {
-  recommended: require('../config/recommended')
+  recommended: require('../config/recommended'),
```

Comment:
Unrelated to this PR, should we be exporting the `experimental` config here as well?